appveyor.yml

```diff
@@ -81,6 +81,7 @@ test_script:
   - "cd empty_folder"
   - "SET JOBLIB_MULTIPROCESSING=0"
   - "python -c \"import nose; nose.main()\" -s pystan.tests.test_basic:TestNormal"
+  - "python -c \"import nose; nose.main()\" pystan.tests.test_multinormal_compilation:TestMultiNormalCompilation"
```

Comment:
Now that I think about it. Let's just add a lot of tests (especially those involving ``matrix`` types). Currently the appveyor builds finish in about ~13 minutes. We could probably test a fair amount more without hitting the time limit. We might only need to exclude a few tests rather than explicitly include tests.